src/App.tsx

```diff
@@ -1,12 +1,80 @@
+/* eslint-disable no-console */
 import React from 'react';
 import './App.scss';
+import 'bulma/css/bulma.min.css';
+import { GoodsList } from './components/GoodsList/GoodsList';
+import { LoadingError } from './components/LoadingError/LoadingError';
+import { Loader } from './components/Loader/Loader';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+type State = {
+  goods: Good[] | [];
```

Comment:
No need for an empty arrray type
```suggestion
  goods: Good[];
```